src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,215 @@
+package duke;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+import duke.command.Command;
+import duke.exceptions.DukeEmptyArgumentException;
+import duke.exceptions.DukeException;
+import duke.exceptions.DukeInvalidDateException;
+import duke.exceptions.DukeInvalidIndexException;
+import duke.exceptions.DukeUnknownCommandException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.ToDo;
+import duke.ui.Ui;
```

Comment:
for import statements, maybe you could separate the import statements to different chunks.

e.g. one chunk for duke.exceptions, another for duke.task, etc...

would make the import statements look less daunting

Reply:
I actually did this in my previous commits, but checkstyle suggests to not have extra spaces so I just followed 😅